processing/src/main/java/org/apache/druid/java/util/emitter/service/SegmentMetadataEvent.java

```diff
@@ -104,6 +105,7 @@ public EventMap toMap()
 
     return EventMap.builder()
         .put(FEED, getFeed())
+        .put("timestamp", createdTime.toString())
```

Comment:
It makes sense to set the `timestamp` field, but if we are emitting this with the `SegmentMetadataEvent`, we probably don't need the `createdTime`.

Reply:
The createdTime may already be used by whoever/whatever is consuming this metric. I want to make sure we do not create regression and not remove/modified any existing fields.

Reply:
Yeah, I assumed that you retained it for backwards compat.

But I think it is fine to make a breaking change in this case as I don't think there would be a lot of downstream applications relying on this particular event. At least I have not come across any major use cases. Moreover, as you point out in the description, this event was already behaving differently than the other emitted events. So it's okay to fix that up.

Although, if you feel differently and have come across applications that depend on the `createdTime` field, we can retain it while marking it as deprecated so that we can get rid of it eventually.

Reply:
I have not come across applications that depend on the createdTime field but I am also not the person who added (or even use) this metric haha

Reply:
Cool. In that case, I would advise we get rid of the extra field since I couldn't find in the docs either. So I don't think we are required to have backward compatibility in this case.

But if you want to go ahead with this PR, we can do that later too.